tests/queries/0_stateless/02233_set_enable_with_statement_cte_perf.sql

```diff
@@ -0,0 +1,27 @@
+DROP TABLE IF EXISTS ev;
```

Comment:
@palegre-tiny test is failing during our parallel check, we can disable parallel check or better to update test to support run in parallel. Does this test should work during parallel run ?

Reply:
I can see that all failures were caused by timeouts, the test were too heavy. I have divided by 10 the number of register to make it faster and to be able to run in parallel.
I have also changed the way we set 'enable_global_with_statement' so we set it in the SELECT query instead of 'SET' instruction.